prow/cmd/pipeline/main.go

```diff
@@ -161,14 +162,17 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Failed to create prowjob client")
 	}
-	pjif := prowjobinfo.NewSharedInformerFactory(pjc, 30*time.Minute)
+	pjNamespace := configAgent.Config().ProwJobNamespace
+	pjif := prowjobinfo.NewSharedInformerFactoryWithOptions(pjc, 30*time.Minute,
+		prowjobinfo.WithNamespace(pjNamespace))
 	pjif.Prow().V1().ProwJobs().Lister()
 	go pjif.Start(stop)
 
 	pipelineConfigs := map[string]pipelineConfig{}
 	for context, cfg := range configs {
 		var bc *pipelineConfig
-		bc, err = newPipelineConfig(cfg, stop)
+		pipelineNamespace := configAgent.Config().PodNamespace
```

Comment:
I think the ProwJobNamespace use is not contentious, but this will be. @fejta wants to use the Namespace on the PodSpec for this controller, I think.